### ISSUE TITLE
Availability of other APIs no longer apparent from home page

### ISSUE THREAD

**Issue by drmrbrewer** [open-meteo/open-meteo#308]
The [home page](https://open-meteo.com/) used to have links in a menu item at the top to the various other APIs that are available (e.g. Marine, Geocoding).  But now, these links are gone and it seems to me that the only mention of them is hidden away in the "Other APIs" list at the bottom of the page.  So it's not really that apparent that these other APIs exist.  Maybe that's deliberate, but I thought that I should mention it, in case not!

**Comment by patrick-zippenfenig** [comment]
I moved all APIs to a sidebar navigation to make it a bit easier to navigate between different APIs. The top-navigation dropdown was a bit cumbersome to use. But you are right, that there should be a better overview. Adding a table table with all API on the ["Features" page](https://open-meteo.com/en/features) could be helpful

![Screenshot 2023-04-23 at 13 03 28](https://user-images.githubusercontent.com/11670456/233836188-8131114a-6f7f-439c-9b01-cb3f94c5157e.png)

**Comment by drmrbrewer** [comment]
Yep I saw that there is good visibility of other other APIs once you're on the API docs page, but on the home page itself it would at first sight appear that it's "just" a Weather API.
